src/content/early-years/family-environment.mdx

```diff
@@ -1,6 +1,8 @@
 ---
+order: 2
 parent: Early years
 title: Family environment
+color: aqua
```

Comment:
I think we _technically_ could do that, but that would involve parsing the node contents and also keeping the mapping between the "parent" and color in `gatsby-nodes.js`, and I feel like that mapping doesn't belong there. We could, however, keep that mapping elsewhere, and use it to fetch the color based on parent rather than populating it in the mdx files, I'll give that a try